it/mail/chrome/messenger/messenger.properties

```diff
@@ -485,55 +485,55 @@ copyToFolderAgain=Copia di nuovo in "%1$S"
 copyToFolderAgainAccessKey=o
 
 # Strings for growl notifications on Mac OS X
-subjectNotificationTitle=Subject: "%1$S"
-senderNotificationText=Sender: "%1$S"
-growlNotification=Nuova posta
+subjectNotificationTitle=Oggetto: "%1$S"
+senderNotificationText=Mittente: "%1$S"
+growlNotification=Nuove e-mail
 
 # mailCommands.js
-emptyJunkTitle=Confermare
-emptyJunkMessage=Eliminare definitivamente tutti i messaggi e le sottocartelle nella cartella di posta indesiderata?
+emptyJunkTitle=Conferma
+emptyJunkMessage=Sei sicuro di volere eliminare definitivamente tutti i messaggi e le sottocartelle nella cartella di posta indesiderata?
 emptyJunkDontAsk=Non chiederlo di nuovo.
 
 # nsSearchSuggestions.js
 suggestion_label=Suggerimenti
 
-# searchresults.xml, nsMsgDBView.cpp, msgHdrViewOverlay.js, etc. 
+# searchresults.xml, nsMsgDBView.cpp, msgHdrViewOverlay.js, etc.
 noSubject=(nessun oggetto)
-noRecipient=(no recipient)
+noRecipient=(nessun destinatario)
 
 # pbMsgHdrViewOverlay.js
-showQuotedText=- show quoted text -
-hideQuotedText=- hide quoted text -
+showQuotedText=- mostra testo citato -
+hideQuotedText=- nascondi testo citato -
 #LOCALIZATION NOTE #1 is the number of attachments in the message
-inlineAttachmentTitleForNoAttachments=attachments
-inlineAttachmentTitle1=attachments;#1 attachment;#1 attachments
-inlineSaveAllAttachmentsTitle=save
+inlineAttachmentTitleForNoAttachments=allegati
+inlineAttachmentTitle1=allegati;#1 allegato;#1 allegati
+inlineSaveAllAttachmentsTitle=salva
 notDisplayingRemoteContent.label=Immagini remote non visualizzate.
 loadRemoteImages.label=Carica immagini
 
 # searchresults.xml
 #LOCALIZATION NOTE %1$S is the string being searched on in the search tab
-searchResultsTitle=Search Results for "%1$S"
-searchNoMatchesSuggestionWildcard=Prova ad aggiungere il metacarattere ai tuoi termini di ricerca, come %1$S.
+searchResultsTitle=Cerca risultati per "%1$S"
+searchNoMatchesSuggestionWildcard=Prova ad aggiungere la wildcard ai tuoi termini di ricerca, come %1$S.
 searchTabTitletext=Testo
 searchTabTitleattachments=Allegati
 searchTabTitleimages=Immagini
 searchTabTitleurls=Collegamenti
 searchTabTitlecontacts=Contatti
 searchresults.showMoreResults.load=Carica altri risultati...
 searchresults.showMoreResults.nomore=Nessun altro risultato
-emptySearchResultsNoBody=(no body)
-emptySearchResultsNotIndexed=(this message has not been indexed)
-noSocialStatus=(no status)
-searchRecentMessages=Show %1$S's Messaggi...
-searchRecentAttachments=Show %1$S's Allegati...
-searchRecentImages=Show %1$S's Immagini...
-searchRecentURLs=Show %1$S's Collegamenti...
+emptySearchResultsNoBody=(nessun corpo)
+emptySearchResultsNotIndexed=(questo messaggio non è stato indicizzato)
+noSocialStatus=(nessun stato)
+searchRecentMessages=Mostra %1$S's messaggi...
```

Comment:
CORRECTION NEEDED. 

Here, and in the next entries too, there is a genitive 's after %1$S. Therefore I would translate like this: "Mostra gli allegati di %1$S..."